src/WebApi_ws_console.cpp

```diff
@@ -21,16 +21,30 @@ void WebApiWsConsoleClass::init(AsyncWebServer& server, Scheduler& scheduler)
 
     scheduler.addTask(_wsCleanupTask);
     _wsCleanupTask.enable();
+
+    _simpleDigestAuth.setUsername(AUTH_USERNAME);
+    _simpleDigestAuth.setRealm("console websocket");
```

Comment:
I would suggest to set it to BASIC Auth here and rename it from `_simpleDigestAuth` to `_simpleBasicAuth` for backwards compatibility:

```
# _simpleDigestAuth.setRealm("console websocket");
_simpleBasicAuth.setAuthType(AuthenticationMiddleware::AuthType::AUTH_BASIC);
```